src/libPMacc/include/particles/tasks/TaskSendParticlesExchange.hpp

```diff
@@ -113,6 +113,15 @@ namespace PMacc
         virtual ~TaskSendParticlesExchange()
         {
             notify(this->myId, RECVFINISHED, NULL);
+            if(retryCounter != 0)
+            {
+                std::cerr << "Send/receive buffer for species " <<
```

Comment:
well, we would need std:cerr writing LogLevels and a default lvl for PMacc, so not for now...